near-sdk/src/types/integers.rs

```diff
@@ -0,0 +1,115 @@
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
+
+/// Implements a wrapped integer type that serializes into and deserializes from base10 string.
+macro_rules! impl_str_type {
```

Comment:
Let's name the module to reflect the purpose of these types, e.g. `stringifiable_types` or `str_types`.

Reply:
`serde_types`?

Reply:
Going to use `json_types`